contracts/JBETHPaymentTerminal.sol

```diff
@@ -295,7 +292,7 @@ contract JBETHPaymentTerminal is
         _fundingCycle,
         _distributedAmount,
         _projectOwner,
-        string(bytes.concat('Fee from @', _handle))
+        string(bytes.concat('Fee from ', bytes32(_projectId)))
```

Comment:
Fee from project with ID 1234.  Let's make sure this print out as an int, not gibberish.

Reply:
@peripheralist getting your eyes here. Because we're removing handles from JBProjects, the memos will only show projectIds. The UI should still be able to resolve these to ENS handles. Wonder if there's a standard way you want to signal to the UI to look resolve a projectId into an ens. maybe with an `@` ? i.e. "Fee from @1234" means => lookup project with ID 1234, get the owner address, try to resolve ENS to dispaly.

Reply:
Yeah, I wasn't sure what to do here, and I don't think what I did is great (Ids will probably come out like 0x1ffffff something ugly like that). TBH, I'm not sure what the memos are for, either, would it be a problem to remove them?

Reply:
also cc @odd-amphora. reminds me of the wikitoken memo whoopsies :)

Reply:
@EzraWeller they're great for UI's to show activity: 
![image](https://user-images.githubusercontent.com/77952627/150575201-8c2fa4fc-af57-4fb7-89dc-22dfb4d01355.png)

we can resolve these in the ENS scheme the same way we resolve names for payouts.

Reply:
@EzraWeller take a look at https://github.com/OpenZeppelin/openzeppelin-contracts/blob/b3b83b558ebb9982e27ae5ee0bb5f33f278863dd/contracts/utils/Strings.sol#L15-L35. might be what we need to convert uint256 to string for concating in the string.

Reply:
Yeah, was hoping to avoid having to add that extra strings lib -- would it be terrible to just have the front end figure these memos out based on the events? That seems reasonable to me and fits the general philosophy of minimizing on-chain activity, but I guess this is where we need opinions from the UI folks.

Happy to add the strings lib if yall want to.

Reply:
yeah, need advice from front end folks.

if front end can parse the projectId from the bytes, great.

Reply:
the UI will need to do some string interpolation that will get applied to every memo—even non-fee messages—so `@` may be too generic. but i like the idea of using a standard string that other folks could take advantage of if they wanna reference projects in a memo. maybe just... `@@`?

Reply:
alternatively if we only care about project handle lookups in these memos specifically, the UI could just search for the whole string "Fee from..." "Payout from...", and we don't need a symbol at all.

Reply:
i like the standardization so projects can link to other projects from memos! how bout "jb#1234"

Reply:
I'm wondering why we need the memos at all. I assume we're getting those by watching for events or using a subgraph -- can we just construct a string like "Fee from bla to bla" using the event, without the memo?

Reply:
We could. the way it's currently done is just standardized like all other memos that people can send along with their contributions.

Reply:
i.e. these are all send the `_pay` as memos
![image](https://user-images.githubusercontent.com/77952627/150615544-96ed1e19-50db-4036-bf47-ce61d4cbb3e4.png)

Reply:
I suppose there's not much about the events other than the memo that would give us a special signal that they were issued as a `fee` or `payout`.

Reply:
I have a feeling the bytes version you have up will work fine, we just need to confirm. can we write a simple test to check? cc @dhyon @drgorillamd for extra eyes on this convo

Reply:
The string(bytes.concat('Fee from ', bytes32(_projectId))) is NOT working as intended (uint to bytes32 is ok I guess, but it's then casted to string -> the numeric value ends up as the corresponding ascii (for instance 125 is }  )

uint256 to string should be, as @mejango mentioned handled by OZ lib (or something like https://stackoverflow.com/a/65707309/10974969 )

Reply:
Okay, will add something like this.

Reply:
Updated, hopefully this works better @drgorillamd